package.json

```diff
@@ -1,8 +1,24 @@
 {
   "name": "style-loader",
   "version": "0.16.1",
-  "author": "Tobias Koppers @sokra",
   "description": "style loader module for webpack",
+  "engines": {
+    "node": ">= 0.12.0"
+  },
+  "main": "index.js",
+  "files": [
```

Comment:
I this a rhetoric question just to double check or did I miss something obvious ? 🙃